src/cvode/cvode_impl.h

```diff
@@ -43,6 +43,7 @@ extern "C" {
 /* Shortcuts                                                       */
 /*=================================================================*/
 
+#define CV_SUNCTX cv_mem->cv_sunctx
```

Comment:
I recommend that these three readability macros (`CV_SUNCTX`, `CV_PROFILER` and `CV_LOGGER`) be removed.